src/core/etl/tests/Flow/ETL/Tests/Unit/RowTest.php

```diff
@@ -72,12 +72,6 @@ public function test_getting_schema_from_row() : void
                 Entry::integer('item-id', 1),
                 Entry::string('name', 'one'),
             ),
-            Entry::collection(
-                'tags',
-                new Row\Entries(Entry::integer('item-id', 1), Entry::string('name', 'one')),
-                new Row\Entries(Entry::integer('item-id', 2), Entry::string('name', 'two')),
-                new Row\Entries(Entry::integer('item-id', 3), Entry::string('name', 'three'))
-            ),
```

Comment:
just like above, this should be replaced with another type, ideally Map or Struct

Reply:
We don't have map yet, but the structure is already covered in the test (above this one).

Reply:
yup, that's why it can't be merged now